pxtsim/sound/soundexpression.ts

```diff
@@ -223,14 +224,15 @@ namespace pxsim.codal.music {
         return playing;
     }
 
-    export function __playSoundExpression(notes: string, waitTillDone: boolean): void {
+    export function __playSoundExpression(notes: string, waitTillDone: boolean, volume: number): void {
```

Comment:
Can volume be optional here, like it is in `playSoundExpressionAsync`? This will break builds of any targets that use this function. I know you fixed the microbit side. Is it used anywhere else? Maybe fine if we're confident micro:bit is the only target using this function.

Reply:
Yeah, that's a good point. Based on searches, microbit _should_ be the only thing that uses this, but I'd rather avoid causing problems. I'll change this to be optional.

Reply:
Or rather, make it have a default.